CI/appveyor.install.ps1

```diff
@@ -178,44 +285,48 @@ function InstallLuaModules(){
   StartPart "Installing lua modules"
   Set-Location \LuaRocks
   Step "installing lfs"
-  .\luarocks install LuaFileSystem >> "$logFile" 2>&1
+  exec ".\luarocks" @("install", "LuaFileSystem")
   Step "installing luasql.sqlite3"
-  .\luarocks install LuaSQL-SQLite3 SQLITE_INCDIR="$Env:MINGW_BASE_DIR\include" SQLITE_LIBDIR="$Env:MINGW_BASE_DIR\lib" >> "$logFile" 2>&1
+  exec ".\luarocks" @("install", "LuaSQL-SQLite3", "SQLITE_INCDIR=`"$Env:MINGW_BASE_DIR\include`"", "SQLITE_LIBDIR=`"$Env:MINGW_BASE_DIR\lib`"")
   Step "installing rex.pcre"
-  .\luarocks install lrexlib-pcre PCRE_LIBDIR="$Env:MINGW_BASE_DIR\lib" PCRE_INCDIR="$Env:MINGW_BASE_DIR\include" >> "$logFile" 2>&1
+  exec ".\luarocks" @("install", "lrexlib-pcre", "PCRE_LIBDIR=`"$Env:MINGW_BASE_DIR\lib`"", "PCRE_INCDIR=`"$Env:MINGW_BASE_DIR\include`"")
   Step "installing lua-utf8"
-  .\luarocks install luautf8 >> "$logFile" 2>&1
+  exec ".\luarocks" @("install", "luautf8")
 
   Step "installing luazip"
   Set-Location "$workingBaseDir"
   DownloadFile "https://github.com/rjpcomputing/luazip/archive/master.zip" "luazip.zip"
   ExtractZip "luazip.zip" "luazip"
   Set-Location luazip\luazip-master
   Step "installing luazip"
-  gcc -O2 -c -o src/luazip.o -I"$Env:MINGW_BASE_DIR/include" src/luazip.c >> "$logFile" 2>&1
-  gcc -shared -o zip.dll src/luazip.o -L"$Env:MINGW_BASE_DIR/lib" -lzzip -lz "$Env:MINGW_BASE_DIR/bin/lua51.dll" -lm >> "$logFile" 2>&1
+  exec "gcc" @("-O2", "-c", "-o", "src/luazip.o", "-I`"$Env:MINGW_BASE_DIR/include`"", "src/luazip.c")
+  exec "gcc" @("-shared", "-o", "zip.dll", "src/luazip.o", "-L`"$Env:MINGW_BASE_DIR/lib`"", "-lzzip", "-lz", "`"$Env:MINGW_BASE_DIR/bin/lua51.dll`"", "-lm")
   FinishPart "Installing lua modules"
 }
 
-# This is no longer needed - we selectively pull in the wanted submodules as required:
-# git submodule update --init --recursive
+# install dependencies
 
-if (-not $(Test-Path "$workingBaseDir")) {
-    New-Item "$workingBaseDir" -ItemType "directory"
-}
+CheckAndInstall "7z" "C:\Program Files\7-Zip\7z.exe" { InstallSevenZ }
+CheckAndInstall "cmake" "$CMakePath\cmake.exe" { InstallCmake }
+CheckAndInstall "MSYS" "C:\MinGW\msys\1.0\bin\bash.exe" { InstallMsys }
+CheckAndInstall "Boost" "C:\Libraries\boost_1_60_0\bootstrap.bat" { InstallBoost }
```

Comment:
I think that using `bootstrap.bat` is wrong! It builds the boost libraries when Mudlet actually only wants the template form header files to include in the source code.  This just adds to the build time without benefit AFAICT. Instead we should just copy the `./boost_1_xx_0/boost/` part of the ~100MB download to a place that the Mudlet compilation of `./src/TStar.h` can pull in the required Boost header files...

Reply:
I simply use the `bootstrap.bat` to find out, whether boost is installed there or not. It is not actually called

Reply:
Ah, okay! 😊